rancher2/schema_cluster_rke_config_network.go

```diff
@@ -103,6 +103,12 @@ func clusterRKEConfigNetworkFields() map[string]*schema.Schema {
 				Schema: clusterRKEConfigNetworkWeaveFields(),
 			},
 		},
+		"mtu": {
+			Type:         schema.TypeInt,
+			Optional:     true,
+			Default:      0,
```

Comment:
Why zero? I would have thought 1500 or so.

Reply:
0 is default value on rke to don't change anything, same as in rke provider PR, https://github.com/rancher/terraform-provider-rke/pull/154#discussion_r375316302